### DISCUSSION TITLE
why my loss value is fluctuating

### DISCUSSION THREAD

**Discussion by leechangdong** [haotian-liu/LLaVA#1079] [Q&A]
Hello I found the project interesting and am adding my custom data to try out the new finetuning
I'm using LORA and 7B models due to gpu memory
However, I'm currently training and the loss value is jagged instead of decreasing.

<img width="664" alt="스크린샷 2024-02-05 오후 5 41 02" src="https://github.com/haotian-liu/LLaVA/assets/115126427/ff2f4411-13a6-439e-8f6e-6163924eff87">
<img width="510" alt="스크린샷 2024-02-05 오후 5 40 37" src="https://github.com/haotian-liu/LLaVA/assets/115126427/956eabf2-5400-4545-bef3-1d2888e78fd1">

I am attaching my training script
If anyone has any idea what the problem is, I would appreciate a response. Thank you.

**Comment by ShristiDasBiswas** [comment]
Hi, I am seeing a similar loss log file from my run. Could you please share some insights from your experience on how this can be corrected? Thanks!
